sklearn/linear_model/_ridge.py

```diff
@@ -777,9 +788,9 @@ def _ridge_regression(
                 coef[i] = coef_
             n_iter[i] = n_iter_
 
+        intercept = xp.asarray(intercept)
         if intercept.shape[0] == 1:
             intercept = intercept[0]
-        coef = np.asarray(coef)
```

Comment:
Why remove this line?

Reply:
It's not needed because `coef` is initialized as:

```python
coef = np.empty((y.shape[1], n_features), dtype=X.dtype)
```